src/components/SystemIntakeReview/index.tsx

```diff
@@ -49,7 +82,7 @@ export const SystemIntakeReview = ({
 
   const getSubmissionDate = () => {
     if (status === SystemIntakeStatus.INTAKE_DRAFT) {
-      return 'Not yet submitted';
+      return t('review.notSubmitted');
```

Comment:
This translation is not rendering.  It's super weird because it is correct, could be something local.  I can's even search by this field `notSubmitted` - does return a found match in my IDE/repo.  Changing the key works.  Super strange, does it render correctly on your machine?

Reply:
Good catch! There was a typo in the translation file.